### PR TITLE
HEEDLS-255 - Section Page Add Horizontal Rule

### PR DESCRIPTION
**Pull request by SChandarana**
Horizontal rule added between Diagnostic assessment and tutorials. And another between tutorials and post learning assessments. Logic added to view model so separators are only displayed when necessary, tests have been written for these

### CODE REVIEW

File: DigitalLearningSolutions.Web.Tests/ViewModels/LearningMenu/SectionContentViewModelTests.cs [hunk 1]
```diff
@@ -371,5 +371,109 @@ public void Diagnostic_assessment_completion_status_shows_score_and_attempts_if_
...
+
+            // When
+            var sectionContentViewModel = new SectionContentViewModel(sectionContent, CustomisationId, SectionId);
+
+            // Then
+            sectionContentViewModel.DisplayTutorialSeparator.Should().BeTrue();
+        }
+
+        [Test]
+        public void Tutorial_separator_is_false_if_no_tutorials_exist()
```
Comment (FridaTveit) [inline DigitalLearningSolutions.Web.Tests/ViewModels/LearningMenu/SectionContentViewModelTests.cs:467]: What about if there are tutorials but no post learning assessment? 🙂

**Review by FridaTveit — COMMENTED** [review]
One comment otherwise looks good 🙂

**Review by FridaTveit — APPROVED** [review]
Looks good 🙂

**Comment by FridaTveit** [comment]
@SChandarana this PR now includes a commit from @kwunkiuma, is that intentional?

**Comment by SChandarana** [comment]
Yep, we were just fixing a merge conflict with something he wrote and Frank sorted it

**Pull request merged**